src/common/components/enable-telemetry-setting-description.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+import { privacyStatementText } from '../../content/settings/improve-accessibility-insights';
+import { NamedFC } from '../react/named-fc';
+import { LinkComponentDeps, TelemetryNotice } from './telemetry-notice';
+
+export const EnableTelemetrySettingDescription = NamedFC<LinkComponentDeps>('EnableTelemetrySettingDescription', ({ LinkComponent }) => (
```

Comment:
See note below about `TelemetryNoticeDeps`/`Props`. I think you'd want to do the same here (define `EnableTelemetrySettingDescriptionDeps`/`Props` types). It's just a coincidence for now that this component and that component have the same props and deps. 

Note that the deps object can still just have one `LinkComponent` in it; the fields defined in each deps type should have the same name and type, so they collapse to the same thing.